lib/ProductOpener/Users.pm

```diff
@@ -1073,6 +1076,19 @@ sub add_users_to_org_by_admin ($org_id, $email_list) {
 			# Add the user to the organization
 			add_user_to_org($org_id, $user_id, ["members"]);
 			push @emails_added, $email;
+
+			# send user an email
+			my $user_ref = retrieve_user($user_id);
+			my $template_name = "user_added_to_org.tt.html";
```

Comment:
One thing I was wondering is whether we want that email to be sent by Product Opener or by Odoo?

Reply:
Yes, now that you tell me I believe this might be sent by Odoo actually.